doc/cephadm/adoption.rst

```diff
@@ -211,3 +219,22 @@ Adoption process
 
 #. Check the output of the command ``ceph health detail`` for cephadm warnings
    about stray cluster daemons or hosts that are not yet managed by cephadm.
+
+#. Remove Ceph packages from the host except for ceph-common to keep Ceph commandline tools.
```

Comment:
ceph-common in double backticks?

Reply:
added backticks in commit 08384dbe7f067404523191fbf32c5a91e7241ec1

Reply:
Danke.  The test is now failing because that commit isn't signed-off.  Maybe squash the two?